### ISSUE TITLE
Polynomial Decay Document Page question

### ISSUE THREAD

**Issue by pranavsubramani** [tensorflow/tensorflow#29714]
## URL(s) with the issue:
https://www.tensorflow.org/api_docs/python/tf/train/polynomial_decay

Please provide a link to the documentation entry, for example:
https://www.tensorflow.org/api_docs/python/tf/train/polynomial_decay
## Description of issue (what needs changing):
A clarification of the example and why it actually works.

### Clear description
In the example mentioned, if the global step is 0, it is unclear how the learning rate will actually change.

The formula can be rewritten as follows:

d = (l - e) * (1 - g/s)^p + e

In the example, g = 0, which means the formula becomes (l - e) * 1 + e = l - e + e = l

So, I'm very unsure of why/how the learning rate in this example is actually going to decrease.

**Comment by ymodak** [comment]
```global_step``` is an iteration. Thus when we set it to zero there should be no change since we didn't iterate over the batch. The formula to compute ```polynomial_decay``` does not accept negative values for ```global_step``` which makes sense since we cannot have negative iterations.

**Comment by pranavsubramani** [comment]
If your explanation is correct, I am not sure how the documentation example reflects this:

Example: decay from 0.1 to 0.01 in 10000 steps using sqrt (i.e. power=0.5):

However, when you inspect the code segment, it is very clear that the global step is 0 and remains 0, which implies that you're not actually changing the learning rate. Then how is the example correct?
